packages/components/atoms/f-card/stories/card.stories.js

```diff
@@ -1,41 +1,40 @@
-import {
-    withKnobs, boolean, select, text
-} from '@storybook/addon-knobs';
-import Card from '../src/components/Card.vue';
 import { withA11y } from '@storybook/addon-a11y';
+import Card from '../src/components/Card.vue';
 
 export default {
     title: 'Components/Atoms',
```

Comment:
I think this should be scoped to the package, otherwise it injects the derived or given args into other stories with the same `title` here (or has in my experience).

```suggestion
    title: 'Components/Atoms/f-card',
```

Reply:
Cool – will make sure I do this for any others I update too (as I'm going to convert a few of the packages over to controls and make the stories a little more useful)

Reply:
It looks like if I define this, it does this to the nav – I think it's probably only important when there are multiple stories for the same component?

<img width="253" alt="Screen Shot 2021-02-22 at 17 30 18" src="https://user-images.githubusercontent.com/805184/108746081-c42d0a00-7533-11eb-80b2-3cb66d50af91.png">


Useful to know though, as I'll be splitting the button one up into multiple stories next…

Reply:
I think if you don't define the 'storyName' for an export, it chooses it as the default, and then doesn't render the story as a sub-option in the menu - don't quote me on it though, as I haven't tried this.

Reply:
Ok – I'll have a play around with the conventions and update all the stories at once to reflect the change if this works (otherwise will have some with `storyName` set and others with it not set, if that sounds ok?

Reply:
Sounds good to me